tasks/main.yml

```diff
@@ -25,68 +25,61 @@
 - name: download specified ruby version
   get_url:
     url: "http://ftp.ruby-lang.org/pub/ruby/{{ ruby_version |  }}/ruby-{{ ruby_version }}.tar.gz"
-    dest: /home/vagrant/ruby2.3.3.tar.gz
+    dest: "{{ ruby_version_download_path }}"
     mode: 0644
-    owner: vagrant
-    group: vagrant
-  sudo: yes
-    
-- file:
-    path: "/home/vagrant/ruby2.3.3"
-    state: directory
-    mode: 0755
-    owner: vagrant
-    group: vagrant
+    owner: "{{ owner_name }}"
+    group: "{{ group_name }}"
   sudo: yes
+  sudo_user: "{{ server_user }}"
 
 - name: unpack ruby2.3.3
   unarchive:
-    src: /home/vagrant/ruby2.3.3.tar.gz
+    src: "{{ ruby_version_download_path }}"
     dest: /home/vagrant
   sudo: yes
   sudo_user: "{{ server_user }}"
 
-- name: install ruby2.3.3
+- name: install new ruby version
   command: ./configure
   args:
-    chdir: /home/vagrant/ruby-2.3.3/
+    chdir: /home/vagrant/ruby-"{{ruby_version}}"/
```

Comment:
when using varialbes, the whole line should be between "". like this:
chdir: `"/home/vagrant/ruby-{{ruby_version}}"`

Please fix this for the following lines as well